docs/pages/installation.mdx

```diff
@@ -91,8 +91,8 @@ If you've previously installed Teleport via the APT
 repo at `https://deb.releases.teleport.dev/`, you can upgrade by
 re-running the "Debian/Ubuntu (DEB)" install instructions above.
 
-We will also continue to maintain the legacy APT repo at
-`https://deb.releases.teleport.dev/` for the foreseeable future.
+The legacy APT repo at `https://deb.releases.teleport.dev/` will not receive updates
+past Teleport 14 and will be turned off in after Teleport 14 is no longer supported.
```

Comment:
```suggestion
past Teleport 14 and will be turned off when Teleport 14 is no longer supported.
```

Reply:
Fixed in https://github.com/gravitational/teleport/pull/36069/commits/5e71e7eea19379adf39b0eb77d6e2c20d6cb48bc (though I only dropped the "in")